docker-for-windows/install.md

```diff
@@ -32,6 +32,23 @@ Hyper-V on Windows 10:
     BIOS settings.  For more information, see
     [Virtualization](troubleshoot.md#virtualization-must-be-enabled).
 
+### Windows 10 Home system requirements
+
+ Windows 10 Home machines must meet the following requirements to install Docker Desktop:
```

Comment:
Maybe an introcution like:

Windows 10, version 1903 or higher is capable of running WSL 2. We recommend using the Docker Desktop WSL 2 backend when possible.

Reply:
Looks very good, we now can remove this line and I think we have everything covered between Pro and Home.

Reply:
Yea, I'm wondering if we should also remove the repetition 'system requirements' in the subsection

Reply:
That's a great idea!